### DISCUSSION TITLE
Object member not expanding after creating filter

### DISCUSSION THREAD

**Discussion by n4nisha** [codefori/vscode-ibmi#1241] [General]
<img width="337" alt="image" src="https://user-images.githubusercontent.com/130774219/232100910-5dbeef8e-ff16-41ca-b997-2fec6f046e19.png">
I am interested in using VS code for IBM i. I installed Code for IBM i extension and connected to IBM i server. I changed the current library to the library I usually use in green screen, however I see the library as NOT FOUND and also the object browser is not expanding to view the source physical files. I am able to see the IFS browser though. Please let me know what I am missing here. I have attached the image of filter. 

<img width="367" alt="image" src="https://user-images.githubusercontent.com/130774219/232105390-7acd7172-dbfb-4944-bfca-31be14ab8f3a.png">

**Comment by worksofliam** [comment]
Please share the contents of your Code for IBM i output channel after the list doesn't appear.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Reply by n4nisha** [reply]
/home/HNNIRAD: LC_ALL=EN_US.UTF-8 system "call QSYS/QZDFMDB2 PARM('-d' '-i' '-t')"
SELECT * FROM LEVEL2SQL.O_rPGpIhee
{
    "code": 0,
    "signal": null,
    "stdout": "DB2>",
    "stderr": ""
}

**Reply by n4nisha** [reply]
/home/HNNIRAD: system "DSPFD FILE(LEVEL2SQL/*ALL) TYPE(*ATR) FILEATR(*PF) OUTPUT(*OUTFILE) OUTFILE(LEVEL2SQL/O_s18myN8i)"
{
    "code": 0,
    "signal": null,
    "stdout": "CPF9861: Output file O_S18MYN8I created in library LEVEL2SQL.\nCPF9862: Member O_S18MYN8I added to output file O_S18MYN8I in library LEVEL2SQL.\nCPF3030: 950 records added to member O_S18MYN8I in file O_S18MYN8I in LEVEL2SQL.",
    "stderr": ""
}

**Reply by n4nisha** [reply]
Also please note that I dont have CRTLIB authority with my profile.

I see the below error as well. 

/home/HNNIRAD: system "CRTLIB LIB(LEVEL2SQL) TEXT('Code for i temporary objects. May be cleared.')"
{
    "code": 255,
    "signal": null,
    "stdout": "",
    "stderr": "CPD0032: Not authorized to command CRTLIB in library *LIBL.\nCPF0006: Errors occurred in command."
}

I already have a library LEVEL2SQL in my AS400 box. So I am not sure why the VS code is trying to create one. 

Please help me with this
